vertx-mysql-client/README.adoc

```diff
@@ -29,6 +29,7 @@
 
 MySQL data types are mapped as below.
 
+- BOOLEAN(j.l.Byte)
```

Comment:
why not `java.lang.Boolean` ?

Reply:
Because there is no `BOOLEAN` data type in MySQL, a `BOOLEAN` in MySQL is synonyms for `TINYINT(1)`, so we can only get a byte value for a boolean type, but we don't know whether it's `BOOLEAN` or `TINYINT`. It's more clear stated in the https://dev.mysql.com/doc/refman/8.0/en/numeric-type-overview.html.

Reply:
What we can do the best is store the `TINYINT` value as a byte value, when users directly call `Row#getValue` they get a byte value, when they call `Row#getBoolean` we have an intelligent auto mapping to boolean for them.

Reply:
so we don't support boolean in Tuple for MySQL ? or is it that the natural type mapping in MySQL for TINYINT in `j.l.Byte` but we can still use boolean to read/write TINYINT ?

Reply:
it seems that it is still possible according to code that we can read/write booleans but that the natural type mapping for TINYINT is j.l.Byte, is that right ?

Reply:
@BillyYccc yes indeed and we should document that .

Reply:
yes, when we encode a `boolean`, it's auto transformed to byte 1 or 0 in `TINYINT` format.

Reply:
+1 for documentation, I will add this in documentation later, let's proceed with this first.